### PR TITLE
Fix issues in jaxws-2.2 and jaxws-2.3 bundles

### PR DESCRIPTION
**Pull request by neuwerk**

### CODE REVIEW

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

File: dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/bus/osgi/CXFExtensionBundleListener.java [hunk 1]
```diff
@@ -0,0 +1,202 @@
+/**
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/bus/osgi/CXFExtensionBundleListener.java:1]: What did you change in this file? I suggest commenting:
// Liberty Change Start
...
// Liberty Change End

around the changes, or just
// Liberty Change

for single line changes. That's what we do in Jaxrs

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/org/apache/cxf/binding/soap/SoapMessage.java [hunk 2]
```diff
@@ -34,6 +37,7 @@
     private static final long serialVersionUID = 6310906412722265578L;
     private SoapVersion version;
 
+    private static final Logger LOG = LogUtils.getL7dLogger(SoapMessage.class);
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/org/apache/cxf/binding/soap/SoapMessage.java:40]: // Liberty Change

and maybe an empty line before the constructor

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/org/apache/cxf/binding/soap/interceptor/Soap11FaultOutInterceptor.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
+/**
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/org/apache/cxf/binding/soap/interceptor/Soap11FaultOutInterceptor.java:1]: // Liberty Change

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/org/apache/cxf/binding/soap/saaj/SAAJInInterceptor.java [hunk 4]
```diff
@@ -0,0 +1,329 @@
+/**
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/org/apache/cxf/binding/soap/saaj/SAAJInInterceptor.java:1]: // Liberty Change

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/org/apache/cxf/binding/soap/saaj/SAAJOutInterceptor.java [hunk 5]
```diff
@@ -0,0 +1,230 @@
+/**
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/org/apache/cxf/binding/soap/saaj/SAAJOutInterceptor.java:1]: // Liberty Change

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.2.6.2/bnd.bnd [hunk 6]
```diff
@@ -21,7 +21,7 @@ Include-Resource: \
   @${repo;org.apache.cxf:cxf-rt-ws-policy;${cxfVersion};EXACT}!/META-INF/NOTICE, \
   @${repo;org.apache.cxf:cxf-rt-ws-policy;${cxfVersion};EXACT}!/META-INF/spring.handlers, \
   @${repo;org.apache.cxf:cxf-rt-ws-policy;${cxfVersion};EXACT}!/META-INF/spring.schemas, \
-  @${repo;org.apache.cxf:cxf-rt-ws-policy;${cxfVersion};EXACT}!/org/apache/cxf/**
+  @${repo;org.apache.cxf:cxf-rt-ws-policy;${cxfVersion};EXACT}!/org/apache/cxf/**, \
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.2.6.2/bnd.bnd:24]: ooo, this one's important!

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.2.6.2/src/org/apache/cxf/ws/policy/PolicyDataEngineImpl.java [hunk 7]
```diff
@@ -36,6 +36,10 @@
 import org.apache.cxf.ws.policy.builder.jaxb.JaxbAssertion;
 import org.apache.neethi.Assertion;
 
+
+import com.ibm.websphere.ras.annotation.Trivial;
+
+@Trivial
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.2.6.2/src/org/apache/cxf/ws/policy/PolicyDataEngineImpl.java:42]: // Liberty Change

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.2.6.2/src/org/apache/cxf/ws/policy/PolicyDataEngineImpl.java [hunk 8]
```diff
@@ -113,6 +117,7 @@ private PolicyEngine getPolicyEngine() {
         return policies;
     }
 
+    @Trivial
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.2.6.2/src/org/apache/cxf/ws/policy/PolicyDataEngineImpl.java:120]: // Liberty Change

Comment (WhiteCat22) [reply dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.2.6.2/src/org/apache/cxf/ws/policy/PolicyDataEngineImpl.java:121]: Missed // Liberty Change :P

**Comment by neuwerk** [comment]
#run-libby-bot

**Comment by neuwerk** [comment]
#build

**Pull request merged**